mithril-aggregator/src/http_server/routes/artifact_routes/cardano_stake_distribution.rs

```diff
@@ -108,7 +111,10 @@ pub mod handlers {
         logger: Logger,
         http_message_service: Arc<dyn MessageService>,
     ) -> Result<impl warp::Reply, Infallible> {
-        debug!(logger, "⇄ HTTP SERVER: artifact/epoch/{epoch}");
+        debug!(
+            logger,
+            "GET /artifact/cardano-stake-distributions/epoch/{epoch}"
```

Comment:
```suggestion
            "GET /artifact/cardano-stake-distribution/epoch/{epoch}"
```

Reply:
Log removed in favor of a general route entry log system !